### ISSUE TITLE
[cloud-tensorflow-mnist]:

### ISSUE THREAD

**Issue by kuatroka** [googlecodelabs/feedback#193]
Hi, the material is pretty great. Very interestingly presented and the key points highlighted nicely. I would suggest two things:
1. Make an explicit statement that the tensorflowvisu is a file, not a python module and that the code has to be run from within the cloned repository. That copy/paste of the code won't work.

2. Some link to interpretation of possible Cross-entropy and Acuracy curves. For example:
  i. If the Test Cross-entropy is bowshaped up and Train Cross-entropy is bowshaped down, it means Learning rate is too high/too low or batch-size is too high/too low...etc

Especially point 2 would greatly help the coders (not Phds) to understand the dynamic we need to follow in order to improve the test results accuracy.
